### ISSUE TITLE
staticcheck: ST1005 (problems with error strings)

### ISSUE THREAD

**Issue by timburks** [apigee/registry#744]
https://staticcheck.io/docs/checks#ST1005

Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use `fmt.Errorf("something bad")` not `fmt.Errorf("Something bad")`, so that `log.Printf("Reading %s: %v", filename, err)` formats without a spurious capital letter mid-message.

```
$ staticcheck ./... | grep ST1005 > STATICCHECK
$ wc -l STATICCHECK 
69 STATICCHECK
$ cat STATICCHECK 
cmd/registry/cmd/auth/print_token.go:36:13: error strings should not be capitalized (ST1005)
cmd/registry/cmd/auth/print_token.go:36:13: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/auth/print_token.go:46:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/auth/print_token.go:46:12: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/configurations/activate.go:39:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/activate.go:44:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/create.go:41:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/create.go:41:12: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/configurations/create.go:54:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/create.go:60:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/delete.go:51:13: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/delete.go:51:13: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/configurations/delete.go:59:13: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/delete.go:59:13: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/configurations/describe.go:36:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/describe.go:40:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/describe.go:45:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/list.go:41:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/configurations/list.go:46:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/get.go:35:13: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/get.go:35:13: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/get.go:37:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/get.go:46:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/list.go:35:13: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/list.go:35:13: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/list.go:37:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/list.go:42:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/set.go:43:13: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/set.go:43:13: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/set.go:45:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/set.go:54:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/set.go:58:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/unset.go:35:13: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/unset.go:35:13: error strings should not end with punctuation or newlines (ST1005)
cmd/registry/cmd/config/unset.go:37:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/unset.go:41:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/config/unset.go:45:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/export/csv.go:44:12: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api-deployment.go:98:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api-deployment.go:111:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api-spec.go:94:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api-spec.go:107:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api-version.go:90:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api-version.go:103:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api.go:94:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/create-api.go:107:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api-deployment.go:98:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api-deployment.go:111:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api-spec.go:94:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api-spec.go:107:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api-version.go:90:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api-version.go:103:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api.go:94:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/rpc/generated/update-api.go:107:11: error strings should not be capitalized (ST1005)
cmd/registry/cmd/upload/bulk/discovery.go:137:10: error strings should not be capitalized (ST1005)
cmd/registry/cmd/upload/bulk/openapi.go:205:10: error strings should not be capitalized (ST1005)
cmd/registry/cmd/upload/bulk/protos.go:223:10: error strings should not be capitalized (ST1005)
cmd/registry/conformance/conformance-task.go:155:15: error strings should not be capitalized (ST1005)
cmd/registry/conformance/conformance-task.go:167:15: error strings should not be capitalized (ST1005)
cmd/registry/conformance/conformance-task.go:177:15: error strings should not be capitalized (ST1005)
cmd/registry/conformance/conformance-task.go:182:15: error strings should not be capitalized (ST1005)
cmd/registry/conformance/linter.go:78:15: error strings should not be capitalized (ST1005)
cmd/registry/controller/controller.go:217:22: error strings should not be capitalized (ST1005)
cmd/registry/core/print.go:161:10: error strings should not be capitalized (ST1005)
cmd/registry/patterns/parser.go:201:15: error strings should not be capitalized (ST1005)
pkg/config/config.go:35:31: error strings should not be capitalized (ST1005)
pkg/config/config.go:35:31: error strings should not end with punctuation or newlines (ST1005)
pkg/config/config.go:43:28: error strings should not be capitalized (ST1005)
pkg/config/config.go:220:13: error strings should not end with punctuation or newlines (ST1005)
```

**Comment by timburks** [comment]
I think for this we should make one PR to fix all of our own (non-generated) files and keep this open pending a discussion with the https://github.com/googleapis/gapic-generator-go team.

**Comment by theganyo** [comment]
I've addressed all staticcheck internal errors in PR #1093. I feel we should close all other issues related to the staticcheck linter, perhaps aside from one to track the external generate code issue.
